cmd/atlas/atlas.go

```diff
@@ -55,6 +57,25 @@ func loadConfig() error {
 	return nil
 }
 
+func shouldCopyConfig(atlasConfigPath string) bool {
+	// Keep backward compatibility and copy if non-tty
+	if !terminal.IsTerminal(os.Stdout) {
```

Comment:
can you change this to use https://github.com/mongodb/mongodb-atlas-cli/blob/1c845fa25ad07e5c62266a91de8d14c348caca37/internal/terminal/terminal.go#L43 and test if that solves the problem?

Reply:
As we discussed oflfine, we weren't able to identify this without redirecting Stdout or Stderr. Now we were able to test isTerminal works, so we'll simply check Stdout,Stderr and Stdin